lib/simulator-xcode-6.js

```diff
@@ -306,10 +293,10 @@ class SimulatorXcode6 {
     log.debug('Attempting to launch and quit the simulator, to create directory structure');
     log.debug(`Will launch with Safari? ${safari}`);
 
+    await this.run();
+
     if (safari) {
-      await instrumentsUtil.quickLaunch(this.udid, 'com.apple.mobilesafari');
-    } else {
-      await this.run();
+      await this.openUrl('http://www.appium.io');
```

Comment:
Can we open something served by appium, so we don't depend on a working internet connection?